### PR TITLE
Reduce echoheaders image size using nginx-slim image and lua

### PR DESCRIPTION
**Pull request by aledbf**
requires #288

### CODE REVIEW

File: Ingress/echoheaders/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+events {
+    worker_connections 1024;
+}
+
+http {
+    default_type 'text/plain';
+    client_max_body_size 10m;
+
+    server {
+        listen 8080 default_server reuseport;
```
Comment (bprashanth) [inline Ingress/echoheaders/nginx.conf]: can you add oneline comments above nginx config terms (like reuseport, client_max_body_size etc)? Mostly just describe why you're setting it/what happens when we exceed it or it isn't set.

**Comment by bprashanth** [comment]
LGTM but for the comment nit

**Comment by aledbf** [comment]
@bprashanth done

**Comment by bprashanth** [comment]
LGTM

**Pull request merged**